recipes/grass/meta.yaml

```diff
@@ -0,0 +1,101 @@
+{% set version = "7.4.2" %}
+
+package:
+  name: grass
+  version: {{ version }}
+
+source:
+  url: https://grass.osgeo.org/grass74/source/grass-{{ version }}.tar.gz
+  sha256: 18eb19bc0aa4cd7be3f30f79ac83f9d0a29c63657f4c1b05bf4c5d5d57a8f46d
+  patches:
+    - platform.make.in.patch  # [osx]
+    - platform.make.in-linux.patch  # [linux]
+    - loader.py.patch
+    - rules.make.patch  # [osx]
+    - rules.make-linux.patch  # [linux]
+    - aclocal.m4.patch
+    - configure.patch
+    - install.make.patch
+    - module.make.patch  # [osx]
+    - shlib.make.patch  # [osx]
+
+build:
+  number: 0
+  skip: True  # [not osx]
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - pkg-config
+    - m4  # [linux]
+    - bison  # [linux]
+    - flex  # [linux]
+  host:
+    - python <3
+    - numpy
+    - gdal 2.0.*
```

Comment:
Can it work with `2.4` to be consistent with the rest of the `conda-forge` stack? `gdal 2.0` is quite old and the package is probably problematic now due to the migration to the new compilers.

Reply:
Not sure but I'll try.